pkg/services/ngalert/notifier/alertmanager.go

```diff
@@ -381,7 +381,7 @@ func (am *alertmanager) buildReceiverIntegrations(receiver *alertingNotify.APIRe
 }
 
 // PutAlerts receives the alerts and then sends them through the corresponding route based on whenever the alert has a receiver embedded or not
-func (am *alertmanager) PutAlerts(postableAlerts apimodels.PostableAlerts) error {
+func (am *alertmanager) PutAlerts(_ context.Context, postableAlerts apimodels.PostableAlerts) error {
```

Comment:
We don't use the `context.Context` in the internal Alertmanager, but these methods need to accept one to implement the `Alertmanager` interface and be able to swap Alertmanager structs in the future.